rules/windows/process_creation/sysmon_always_install_elevated_msi_spawned_cmd_and_powershell_spawned_processes.yml

```diff
@@ -0,0 +1,36 @@
+title: Always Install Elevated MSI Spawned Cmd And Powershell Spawned Processes
+id: 38cf8340-461b-4857-bf99-23a41f772b18
+description: This rule will looks for Windows Installer service (msiexec.exe) spawned command line and/or powershell that spawned other processes
+status: experimental
+author: Teymur Kheirkhabarov (idea), Mangatas Tondang (rule), oscd.community
+date: 2020/10/13
+references: 
+    - https://image.slidesharecdn.com/kheirkhabarovoffzonefinal-181117201458/95/hunting-for-privilege-escalation-in-windows-environment-50-638.jpg
+tags:
+    - attack.privilege_escalation
+    - attack.t1548.002
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    parent_image:
+        ParentImage|contains: 
+            - '\cmd.exe'
+            - '\powershell.exe'
+    parent_of_parent_image:
+        ParentOfParentImage|contains|all: 
+            - '\Windows\Installer\'
+            - 'msi'
+        ParentOfParentImage|endswith: 
+            - 'tmp'
+    condition: event_id and parent_image and parent_of_parent_image
```

Comment:
Hey @tas-kmanager , nice contribution!

Here you reference an element that is not defined (event_id)

Reply:
Ah i forgot to remove it when i changed it to process_creation instead of Sysmon, thanks for the review! Made the changes just now.

Reply:
It looks ready for me, I think you can mark the review as resolved.

Reply:
awesome collab, guys!